### ISSUE TITLE
Blank white screen when using 0.3.0-build139 

### ISSUE THREAD

**Issue by IlyaGulya** [JetBrains/compose-jb#263]
A simple example led to the white screen instead of the application UI.
When I've switched back to 0.3.0-build135 everything became normal.
The sample project is here: https://github.com/IlyaGulya/bitwarden-client-mpp/tree/bug-compose-jb-0.3.0-build139

How to run project:
```
./gradlew desktopApp:run
```

Jetpack compose plugins version can be changed in the root `build.gradle.kts`

Environment:
```
$ java -version
openjdk version "11.0.9.1" 2020-11-04
OpenJDK Runtime Environment (build 11.0.9.1+1-Ubuntu-0ubuntu1.20.04)
OpenJDK 64-Bit Server VM (build 11.0.9.1+1-Ubuntu-0ubuntu1.20.04, mixed mode, sharing)
```

Here's `0.3.0-build139` result:
![image](https://user-images.githubusercontent.com/668727/104104654-8002e480-52ba-11eb-9db1-58e8317a852f.png)
Here's `0.3.0-build035` result:
![image](https://user-images.githubusercontent.com/668727/104104715-db34d700-52ba-11eb-9de3-24c4d1b044a5.png)

**Comment by olonho** [comment]
Could you please provide details on what is your graphical card and drivers?

**Comment by olonho** [comment]
Unless additional info could be provided, we have to close the issue. Please reopen when more info available (checked reproducer on my Ubuntu 20.10, and works as expected).

**Comment by IlyaGulya** [comment]
Here's more details:
I'm on KDE Neon User Edition 5.20 (Ubuntu 20.04)
GPU: GeForce GTX 1650
Video driver:  Proprietary NVIDIA driver v.455.38
Linux kernel: 5.6.0-1034-oem
OpenGL Info:
```
$ glxinfo | grep OpenGL
OpenGL vendor string: NVIDIA Corporation
OpenGL renderer string: GeForce GTX 1650/PCIe/SSE2
OpenGL core profile version string: 4.6.0 NVIDIA 455.38
OpenGL core profile shading language version string: 4.60 NVIDIA
OpenGL core profile context flags: (none)
OpenGL core profile profile mask: core profile
OpenGL core profile extensions:
OpenGL version string: 4.6.0 NVIDIA 455.38
OpenGL shading language version string: 4.60 NVIDIA
OpenGL context flags: (none)
OpenGL profile mask: (none)
OpenGL extensions:
OpenGL ES profile version string: OpenGL ES 3.2 NVIDIA 455.38
OpenGL ES profile shading language version string: OpenGL ES GLSL ES 3.20
```

**Comment by IlyaGulya** [comment]
@olonho I can't reopen this issue myself

**Comment by igordmn** [comment]
I investigated this issue, and discovered that we have white screen on 0.3.0-build138 with 200% scale (Ubuntu). On 0.3.0-build138 everything is ok.

**Comment by IlyaGulya** [comment]
Yeah, I have 150% scale

**Comment by igordmn** [comment]
Probably should be fixed soon.

Can you check if the issue still exists on 0.0.0-vsync-build30?

**Comment by IlyaGulya** [comment]
Yep, `0.0.0-vsync-build30` works fine :)

**Comment by igordmn** [comment]
Should be fixed in 0.3.0-build146.

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-5549">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
